radio/src/targets/horus/CMakeLists.txt

```diff
@@ -85,7 +85,19 @@ if (PCB STREQUAL X10)
     add_definitions(-DRADIO_FAMILY_T16)
     option(INTERNAL_MODULE_MULTI "Support for MULTI internal module" ON)
     option(BLUETOOTH "Support for bluetooth module" OFF)
+    set(HARDWARE_TOUCH YES)
```

Comment:
T18 doesn't have working touch

Reply:
Shit sorry about the goof up there, I was working on adding TX18S support and finally figured out what was wrong with my builds.

I also "fixed" the fromSet deprecation, but apparently the QT libs on travis still don't have those changes so that needs to be reverted as well.

Is there a means of leaving a commit out of a PR? Or else Ill just rebase on the latest master.

Reply:
Don't think your can exclude a commit, but you can just revert them. Or if you want to make it clean rename your branch, recreate one with the same name and cherry pick the commits you want, as long as the branch name is the same it'll still link to the PR